component/src/test/resources/security/sp.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
remove this file

Reply:
This is a dummy file with wrong credentials.